components/hab/src/command/studio/docker.rs

```diff
@@ -71,6 +73,9 @@ pub fn start_docker_studio(_ui: &mut UI, args: &[OsString]) -> Result<()> {
         volumes.push(format!("{}:{}/{}",
                              cache_artifact_path, mnt_prefix, CACHE_ARTIFACT_PATH));
     }
+    if let Ok(cache_ssl_path) = henv::var(SSL_PATH_ENVVAR) {
+        volumes.push(format!("{}:{}/{}", cache_ssl_path, mnt_prefix, CACHE_SSL_PATH));
```

Comment:
Instead of "{}/{}", should we use `Path::join` instead for cross-platform pathing (both here and above)?

Reply:
That seems reasonable, however I'm trying to make the minimum change possible in order to minimize the risks of additional refactoring.

Reply:
If I am remembering correctly, `\` can cause problems in docker mounts.

Reply:
If that's the case, it would probably help to document that fact and make a function which takes care of this so that some later well-intentioned person doesn't break things later.

Reply:
Cool, I'll add a comment for now

Reply:
before adding the comment we should verify. I could be misremembering or it is also possible docker has fixed this in recent versions.

Reply:
Either I misremembered or (more likely in this case) docker has fixed this because I just validated `\` works fine in volume mounts.

Reply:
Still probably safer to leave it as is than risk some regression scenario